src/backend/highlighter.js

```diff
@@ -16,7 +29,10 @@ export function highlight (instance) {
   if (!instance) return
   const rect = getInstanceRect(instance)
   if (rect) {
-    showOverlay(rect)
+    let content = ''
+    const name = getInstanceName(instance)
+    name && (content = `<span style="opacity: .6;">&lt;</span>${name}<span style="opacity: .6;">&gt;</span>`)
```

Comment:
I would replace this with

```js
if (name) {
	content = ...
	showOverlay(...)
}
```
since it doesn't look like it's displaying anything otherwise

Reply:
It can still show the component boundaries even if the component doesn't have a name (most certainly using an old version of vue-loader or vue).